cloudnatix/manifest.json

```diff
@@ -0,0 +1,76 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "caebfbe2-48ba-443f-b2c6-bd80122b2605",
+  "app_id": "cloudnatix",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Integration with CloudNatix",
+    "title": "CloudNatix",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "dashboard",
+        "image_url": "images/cloudnatix-dashboard.png"
+      },
+      {
+        "media_type": "image",
```

Comment:
The last two images are quite similar, I'd recommend removing one.

Reply:
Sure! I've removed the third image.